tests/test_formatter.py

```diff
@@ -104,6 +108,8 @@ def test_hide_nested(self):
         output = list(ext._format_command(ctx, show_nested=False))
 
         self.assertEqual(textwrap.dedent("""
+        A sample command group.
```

Comment:
I broke this in 569e4e943ceb5e54fe685976cb75c2dbbfb6c0b5 and never noticed. Time to setup Travis, I think